guides/distributed-run.md

```diff
@@ -11,8 +11,9 @@ and start scenarios on all known nodes (except master).
 ```erlang
 amoc_dist:do(my_scenario, 100, Settings).
 ```
+
 ```elixir
-:amoc_dist.do(:my_scenario, 100, settings).
+:amoc_dist.do(:my_scenario, 100, settings)
```

Comment:
This is the main reason my elixir code doesn't compile, those accidental commas I'm so used to! 😅